app/components/Settings/scheduler/create-modal.jsx

```diff
@@ -104,6 +104,9 @@ class CreateModal extends Component {
         if (!this.state.interval) {
             return this.setState({error: 'Please select an interval above.'});
         }
+        if (this.state.name && this.state.name.trim().length === 0) {
```

Comment:
I haven't debugged the reason, but this isn't working for me. I'm still able to create queries without a name:

![peek 2018-07-26 10-50](https://user-images.githubusercontent.com/6199391/43255486-dca2c498-90c1-11e8-9469-8a239a0fd030.gif)

Reply:
That’s by design I think...

Reply:
Er... perhaps not, looking at the code. I should have said “I had assumed it was by design that you can create queries without names and it didn’t bother me”

Reply:
Well, it looks like you can create queries named `""` but not `"   "`... The first part of the `if` condition is falsey for `""`. Given that we still have to support unnamed queries, we may as well just trim and move on if the user tries `"   "` instead of complaining :)

Reply:
@briandennis How about `if ((this.state.name || '').trim().length === 0)?`

Reply:
My preference here would be to allow any trimmed string, even empty ones, but I'm also OK with the current behaviour if we're out of time.